.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature Request
+about: Help us make Upbound more useful
+labels: enhancement, needs-project, needs-points-label, needs-epic-link
+---
+<!--
+Thank you for helping to improve Upbound! We use issues for bug reports and
+feature requests. Please be sure to search for open issues before raising a new
+one.
+-->
+
+### What problem are you facing?
+<!--
+Please tell us a little about your use case - it's okay if it's hypothetical!
+Leading with this context helps frame the feature request so we can ensure we
+implement it sensibly.
+--->
+
+### How could Upbound help solve your problem?
+<!--
+Let us know how you think Upbound could help with your use case. 
+-->
```

Comment:
@negz What if we added the following checklist items as well to help with our process:

- [ ] I have linked to a parent issue or feature on our roadmap (optional)

Reply:
Yeah, that was the original plan. I had hoped to automate that process using GitHub Actions. Sadly based on my investigation there isn't much we can do about that when the epics are in different repos.